homeassistant/components/nextdns/binary_sensor.py

```diff
@@ -77,7 +74,7 @@ class NextDnsBinarySensor(
 
     def __init__(
         self,
-        coordinator: NextDnsConnectionUpdateCoordinator,
+        coordinator: NextDnsUpdateCoordinator[ConnectionStatus],
         description: NextDnsBinarySensorEntityDescription,
```

Comment:
Not directly related to this PR but it looks like `NextDnsBinarySensorEntityDescription` doesn't need to be generic since it's always `ConnectionStatus`.

Reply:
Thanks for the suggestion, I'll address it in the next PR.

Reply:
https://github.com/home-assistant/core/pull/116697